### ISSUE TITLE
Ratio Highlights

### ISSUE THREAD

**Issue by knightingale242** [easeghehey/Ratio-bot#18]
we can retweet or post the biggest ratios we have seen in an interval of time whether it be daily or weekly

**Comment by pratuln** [comment]
this will be much easier than trying to implement screenshot idea

**Comment by easeghehey** [comment]
must make db

**Comment by knightingale242** [comment]
we can use something lightweight like sqlite embedded into the python application maybe? mysql database seems like over kill

**Comment by easeghehey** [comment]
agree with sqlite

could have 2 tables. 1 for top ratioers, 1 for ratios we find.

i suggest this schema:
Stats(userID: PK, ratioWins, ratioL, ratioFound)
DetectedRatios(mentionID: PK, ratioTweetID, ratio'dTweetID, likeCount) **_we need all 3 IDs to identify the tweets later and in the case a tweet gets deleted etc...**

the Stats table must get reset every month (or maybe week). and it will be used by users to access their stats
Detected Ratios will be for us to post the top ratio every week (maybe month but tweets get deleted/suspended...) and then we reset that table to save processing power

**Comment by knightingale242** [comment]
Agreed, I like this schema. This solves a problem I was thinking about as far as having both of those functionalities.
